src/ui-manager.js

```diff
@@ -170,4 +171,16 @@ export default class UIManager {
   setLogLevel(level: Object, name?: string) {
     setLogLevel(level, name);
   }
+
+  /**
+   * Handle forcing touch ui
+   * @returns {void}
+   */
+  _handleTouchUi(): void {
```

Comment:
I think this should be a generic method which handles all the possible UI config and not a specific config (similar to `handlePlaybackConfig` in playkit.
So, change the name to be more generic.